.github/workflows/tests.yml

```diff
@@ -69,7 +71,8 @@ jobs:
       - name: Lint and Type Check
         run: |
           npm run lint
-          npm run build:types
+          # TODO: typecheck fails if there is no build, see if tests are importing from lib
+          npm run build:node:cjs
```

Comment:
I tried to fix the issue (tests import from src using path aliaseses) in this commit:
https://github.com/jsforce/jsforce/pull/1369/commits/7a4c6c9d195e70bfc444b1d5235670748b52cabc

put [W-14321329](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001ch8scYAA/view) in the jsforce epic to fix later